### PR TITLE
update release docs

### PR DESCRIPTION
**Pull request by nsa-brant**

### CODE REVIEW

File: docs/RELEASE_HOW_TO.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+2. **Commit the Changes:**
+   - After updating the changelog, commit your changes. Make sure your commit message is clear and descriptive.
+   - Example commit message:
+     ```
+     Update changelog for version 8.8.8
+     ```
+
+3. **Tag the Commit:**
+   - Tag the commit including or afteryour changelog update. This tag will trigger the release workflow. Pythonik uses semantic versioning. Please see the [semver.org](https://semver.org/) website for more information.
```
Comment (nsa-prince) [inline docs/RELEASE_HOW_TO.md:25]: Forgot a space here. I assume you meant to type
> after your

**Review by nsa-prince — CHANGES_REQUESTED** [review]
There's a typo in the release doc

File: docs/RELEASE_HOW_TO.md (hunk 1 above)
Comment (nsa-brant) [reply docs/RELEASE_HOW_TO.md]: Thanks! Fixed

**Comment by nsa-brant** [comment]
Thanks fixed the typo

**Review by nsa-prince — APPROVED** [review]
💯

**Pull request merged**